lib/experimental/fonts/font-library/class-wp-rest-font-collections-controller.php

```diff
@@ -57,6 +59,7 @@ public function register_routes() {
 					'callback'            => array( $this, 'get_item' ),
 					'permission_callback' => array( $this, 'get_items_permissions_check' ),
 				),
```

Comment:
Need to add the context param to this route.

```suggestion
					'args'                => array(
						'context' => $this->get_context_param( array( 'default' => 'edit' ) ),
					),
				),
```

Reply:
added here but defaulting to `view` instead of `edit`: https://github.com/WordPress/gutenberg/pull/58222/commits/a0e5d722a60949b1c978cc536ec25d6189bcc0a1